presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -2188,6 +2188,10 @@ public void testLargeIn()
                 .collect(joining(", "));
         assertQuery("SELECT orderkey FROM orders WHERE orderkey IN (" + longValues + ")");
         assertQuery("SELECT orderkey FROM orders WHERE orderkey NOT IN (" + longValues + ")");
+        assertQuery("SELECT orderkey FROM orders WHERE CAST(orderkey AS DECIMAL(38)) IN (" + longValues + ")");
```

Comment:
These tests are ineffective, at least for `decimal(38)`, since it is converted to `bigint` during planning, which I verified by looking at the explain plan. I'm not sure why it happens for `decimal(38)` but not `decimal(18)`.

Let's remove these and replace with direct tests for `InCodeGenerator`, as we don't need to run such expensive tests against every connector. The primary purpose of this test is to very end-to-end against the engine and connectors for large sets. (if we only care about `InCodeGenerator`, we just need a list of size `> 32`)